framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -764,11 +764,9 @@ public AnnotationMirror getAnnotationFromJavaExpressionString(
      * @param tree current tree
      * @param clazz the Class of the annotation
      * @return the annotation on expression or null if one does not exist
-     * @throws FlowExpressionParseException thrown if the expression cannot be parsed
      */
     public AnnotationMirror getAnnotationFromReceiver(
-            FlowExpressions.Receiver receiver, Tree tree, Class<? extends Annotation> clazz)
```

Comment:
Why the change to `GenericAnnotatedTypeFactory`? This seems unrelated to the rest of the change.